spec/swagger.yaml

```diff
@@ -5674,7 +5745,66 @@ definitions:
         type: string
         description: The refresh token returned from a previous authentication request.
 
-        
+  AccountAPIProjectListSuccessResponse:
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          required:
+            - totalCount
+            - items
+          properties:
+            totalCount:
+              type: integer
+              description: The number of projects that match your specified criteria
+            items:
+              type: array
+              items:
+                $ref: '#/definitions/AccountAPIListProjectItem'
+
+  AccountAPIListProjectItem:
+    type: object
+    required:
+      - projectId
+      - projectName
+      - accountUid
+      - archived
+      - projectTypeCode
+      - sourceLocaleId
+      - sourceLocaleDescription
+    properties:
+      projectId:
+        type: string
+        description: A unique identifier for the project
+      projectName:
+        type: string
+        description: The name of the project. Note, names of projects are not guarenteed to be unique within an Account.
```

Comment:
Note: Names of projects are not guaranteed to be unique within an account.

Reply:
done